src/ktn_code.erl

```diff
@@ -746,7 +776,7 @@ macro_name(Name) ->
 %%      <code>
 %%        split_when(fun (X) -> $. == X end, "a.b.c") = ["a.", "b.", "c"]
 %%      </code>
-%%      NOTE: Copied from ktn_lists not to bring the whole erlang-katana
+%%      NOTE: Copied from ktn_lists not to bring the whole Erlang-katana
```

Comment:
The repo name is written with lowercase, it shouldn't have been capitalized.

Reply:
7e4ec84